src/ReferenceList.php

```diff
@@ -241,4 +245,59 @@ public function unserialize( $data ) {
 		$this->__construct( unserialize( $data ) );
 	}
 
+	/**
+	 * Removes duplicates bases on hash value.
+	 *
+	 * @since 0.2
+	 */
+	public function removeDuplicates() {
```

Comment:
No, we should remove this method (and many others). However, I'm not sure if our next release should be a breaking release again. :S